.github/workflows/ci-dgraph-load-tests.yml

```diff
@@ -0,0 +1,71 @@
+name: ci-dgraph-load-tests
+on:
+  push:
+    branches:
+      - main
+      - release/*
```

Comment:
@joshua-goldstein are you sure about load tests? I was under the assumption that this will not work. You are better off enabling `coverage`.

Reply:
I have removed `release/*` from the load test workflow and enabled coverage.